docs_src/getting-started/quick-start/index.md

```diff
@@ -1,227 +1,255 @@
 
+!!! Note
+    The above is the Geneva release of EdgeX for use on **Intel x86** platforms.  If you are running on ARM, use this [docker-compose file instead](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis-no-secty-arm64.yml).
 
-Once you have Docker and Docker Compose installed, you need the configuration file for downloading and running the EdgeX Foundry docker containers. Download the latest `docker-compose` file here <https://github.com/edgexfoundry/developer-scripts/raw/master/releases/fuji/compose-files/docker-compose-fuji.yml> and save this as `docker-compose.yml` in your local directory. This file contains everything you need to deploy EdgeX with docker.
-
-First, use this command to download the EdgeX Foundry Docker images from Docker Hub:
+First, use this command to download the EdgeX Foundry Docker images from Docker Hub.  From a terminal window, change directory to your local directory containing the docker-compose.yml and execute the following Docker Compose command:
 ```
 docker-compose pull
 ```
-Then start up all of the EdgeX Foundry microservices:
+This fetches all the referenced EdgeX containers.  Then start up all of the EdgeX Foundry microservices:
 ```
 docker-compose up -d
 ```
-Finally, verify that the EdgeX containers have all been started:
+The -d option on docker-compose up is to start the containers in detached mode which allows the containers to run independently from the terminal window.  Finally, verify that the EdgeX containers have started:
 ```
 docker-compose ps 
 ```
-!!! Note 
-    Initialization or seed containers, like config-seed, will have exited as their job is just to initialize the associated service and then exit.
-
 ![image](EdgeX_GettingStartedUsrActiveContainers.png)
-
-The following table captured the Default Service Ports. This table provides the default ports used by each of the EdgeX micro services (per its default configuration and the EdgeX provided docker-compose files).  These default ports are also used in the EdgeX provided service routes defined in the Kong API Gateway for access control.
-
-
-|Services Name|	Port Definition|
-|---|---|
-|consul	|8400|
-| 	|8500|
-| 	|8600|
-|vault	|8200|
-|kong-db	|5432|
-|kong	|8000|
-| 	|8001|
-| |	8443|
-| 	|8444|
-|mongo|	27017|
-|redis|	6379|
-|system	|48090|
-|core-data|	48080|
-| 	|5563|
-|core-metadata	|48001|
-|core-command	|48082|
-|support-notifications	|48060|
-|support-logging	|48061|
-|support-scheduler|	48085|
-|device-virtual	|49990|
-|device-random	|49988|
-|device-mqtt	|49982|
-|device-modbus	|49991|
-|device-snmp	|49993|
-|UI	|4000|
-
+*If all EdgeX containers pulled and started correctly and without error, you should see a process status (ps) that looks similar to the image above.*
 
 ## Connecting a Device
```

Comment:
I think we should a put break here and maybe add a couple accordions for a device section and a app service section. Allowing the user what step they take next.

Reply:
broke out to a REFERENCE section so we can move it later, and put each of the service types under a category in a collection of tabs.